app/src/main/java/org/oppia/android/app/options/ReadingTextSizeSelectionViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package org.oppia.android.app.options
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.MutableLiveData
+import org.oppia.android.R
+import org.oppia.android.app.fragment.FragmentScope
+import org.oppia.android.app.model.ReadingTextSize
+import org.oppia.android.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/**
+ * Text Size list view model for the recycler view in [ReadingTextSizeFragment]
```

Comment:
Update to `/** Text Size list view model for the recycler view in [ReadingTextSizeFragment]. */`